bolt-modules/boltlib/spec/functions/file_upload_spec.rb

```diff
@@ -23,7 +23,7 @@
     let(:message) { 'uploaded' }
     let(:result) { Bolt::Result.new(target, message: message) }
     let(:result_set) { Bolt::ResultSet.new([result]) }
-    let(:module_root) { File.expand_path(fixtures('modules', 'test')) }
+    let(:module_root) { File.expand_path(fixtures('bolt-modules', 'test')) }
```

Comment:
I don't think we need this isn't fixtures/modules created by rspec puppet

Reply:
I'm not exactly sure what the replacement needs to be. We use module_root in the next line.

Reply:
You should be able to just undo this change. a helper will create spec/fixtures/modules is the correct path.

Reply:
Ah, this path is setup by rspec-puppet. Not sure how to access it yet.

Reply:
Done.